pkg/fission-cli/function.go

```diff
@@ -224,6 +224,7 @@ func fnCreate(c *cli.Context) error {
 		}
 	}
 	entrypoint := c.String("entrypoint")
+	fnTimeout := c.Int("fntimeout")
```

Comment:
Return error when a user specifies the illegal value, just like how we do in specialization timeout.
https://github.com/fission/fission/blob/e481c234ee038f8ce8c877a4ab7d3fbf8b8b5597/pkg/fission-cli/function.go#L155-L160